src/util/cosmos/iscn.js

```diff
@@ -54,8 +54,8 @@ export async function getISCNSigningAddressInfo() {
 }
 
 export function getISCNPrefix(input) {
-  const res = /^(iscn:\/\/likecoin-chain\/[A-Za-z0-9-_]+)\/[0-9]+$/.exec(input);
-  if (!res) return input;
+  const res = /^(iscn:\/\/likecoin-chain\/[A-Za-z0-9-_]+)(?:\/?([0-9]*))?$/.exec(input);
```

Comment:
```suggestion
  const res = /^(iscn:\/\/likecoin-chain\/[A-Za-z0-9-_]+)(?:\/([0-9]+))?$/.exec(input);
```

Reference: the ISCN ID regex pattern in LikeCoin chain source code:
```Go
const IscnIdRegexpPattern = "iscn://([-_.:=+,a-zA-Z0-9]+)/([-_.:=+,a-zA-Z0-9]+)(?:/([0-9]+))?$"
```